frontend/containers/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -249,7 +315,7 @@ class DocumentEditToolbar extends Component {
       collection,
       group,
       referencedField,
-      state
+      state,
```

Comment:
🤔

Reply:
that's a weird one that crept in refactors, needs a good eslint